src/engraving/rw/read206/read206.cpp

```diff
@@ -2045,6 +2045,12 @@ static void readVolta206(XmlReader& e, ReadContext& ctx, Volta* volta)
 
 static void readPedal(XmlReader& e, ReadContext& ctx, Pedal* pedal)
 {
+    if (pedal->score()->mscVersion() < 420) {
+        pedal->setBeginText(String());
+        pedal->setContinueText(String());
+        pedal->setEndText(String());
```

Comment:
No, I don't think so? if there's no tag present, doesn't that mean the property should follow the default?  If I do set them to unstyled, then the start and end text is set based on hook/line visibility but the continue text is always read as a blank string which is incorrect.

Reply:
If no tag is present, the property should indeed follow the default; so in principle, it should be STYLED indeed. But in this case, the default has changed; so if no tag is present in an old score, that means that it should follow the _old_ default, namely an empty string. It looks indeed like your code is properly doing that.

But if you set them to an empty string at the start of the method, and don't set them to UNSTYLED, then you basically get in an inconsistent state, because the property value (which is an empty string) does not equal the style value (which is now some text), while having its flags set to STYLED implies that it should be equal. That's what I'm wondering about.

Reply:
Ah sorry, I understand now thanks!  Yes that makes sense, they should be set to UNSTYLED